pkg/transformerconfig/pathconfig.go

```diff
@@ -32,5 +32,14 @@ type PathConfig struct {
 
 // PathSlice converts the path string to a slice of strings, separated by "/"
 func (p PathConfig) PathSlice() []string {
-	return strings.Split(p.Path, "/")
+	if !strings.Contains(p.Path, "\\/") {
```

Comment:
"/" can be contained in a fieldname. such as `ingress.kubernetes.io/auth-secret` in Ingress annotations.
To deal with this special case, the path to this field should be formatted as 
```
metadata/annotations/ingress.kubernetes.io\/auth-secret
```
Then `PathSlice` will return
```
[]string{"metadata", "annotations", "ingress.auth-secretkubernetes.io/auth-secret"}
```

Reply:
Nice descriptions.   Might as well add it to the method's comments.

Reply:
I'll add it.